### PR TITLE
Create class school library

### PR DESCRIPTION
**Pull request by getssh**
Hello, I have worked on this project

- Created Person class with instance variables of @id, @name, and @age
- Create a constructor with name, age, and parent_permission as parameters. name and parent_permission are optional
- Getters for @id, @name, and @age.
- Setters for @name and @age.
- Private method of_age?
- Create class Student Inherits from Person
- Create class Teacher Inherits from Person
- Add readme file
- Checked for syntax errors using Rubocop

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  def of_age?
+    return true if @age >= 18
+
+    false
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```
Comment (mikethreels) [inline person.rb]: - [x] everything below the keyword `private` is a private method, this means that in this setup the can_use_services? method is private as well which it shouldn't be in order for this to work, please adjust this so the can_use_services? method is public.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Getayawkal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job setting up the associations.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (getssh) [reply person.rb]: Thank you for the feedback, I have fixed the code according to the requested change

**Review by BregornOriginal — APPROVED** [review]
Hi @getssh 

Your project is complete!  There is nothing else to say other than... it's time to merge it 

Congratulations! 

giphy

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good work implementing changes requested by the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**